lib/components/text/serif.js

```diff
@@ -7,7 +7,7 @@ const { StyleSheet, Text } = React;
 export default class Serif extends React.Component {
   render() {
     return (
-      <Text style={[styles.default, this.props.style, styles.required]}>
+      <Text style={[styles.default, this.props.style, styles.required]} numberOfLines={1}>
```

Comment:
This has to happen here - it won't work if you try adding it to a SerifText instance elsewhere.